rllib/offline/estimators/tests/gridworld.py

```diff
@@ -0,0 +1,143 @@
+from ray.rllib.policy.policy import Policy, ViewRequirement
```

Comment:
This file should go into `rllib.examples.env`

Reply:
The policies should go into `rllib.examples.policy.gridworld_deterministc_policies`

Reply:
Maybe instead of gridworld, make the name more explicit like cliff_walking_custom or something similar.